UnofficialSteamAuthenticator/App.xaml.cs

```diff
@@ -14,9 +14,9 @@ namespace UnofficialSteamAuthenticator
     /// <summary>
     /// Provides application-specific behavior to supplement the default Application class.
     /// </summary>
-    public sealed partial class App : Application
```

Comment:
It's not a child class. It's a **partial** class.

But I don't care about it really, just removed it for simplicity because `App` will **always** inherit from `Application`.